test_common/harness/imageHelpers.cpp

```diff
@@ -1809,7 +1806,7 @@ static float unnormalize_coordinate(const char *name, float coord, float offset,
     switch (addressing_mode)
     {
         case CL_ADDRESS_REPEAT:
-            ret = RepeatNormalizedAddressFn(coord, static_cast<size_t>(extent));
+            ret = RepeatNormalizedAddressFn(coord, static_cast<size_t>( extent) );
```

Comment:
Are these diffs actually required?
I understand you're changing code styling here, but it doesn't seem to be necessary for this change, and it is inconsistent with other invocations of the static_cast<> elsewhere on the patch.